docs/user-guide/building-collection.md

```diff
@@ -0,0 +1,86 @@
+# Building Ansible Collection using with Unified Tooling
+
+This guide illustrates a comprehensive Ansible development workflow that showcases the integration of various tools within the Ansible ecosystem to create an Ansible collection. The tools featured in this workflow include:
+
+- [ansible-creator](https://github.com/ansible/ansible-creator)
+- [ansible-development-environment (pip4a)](https://github.com/ansible/ansible-development-environment)
+- [ansible-lint](https://github.com/ansible/ansible-lint)
+- [ansible-navigator](https://github.com/ansible/ansible-navigator)
+- [VS Code Ansible extension](https://github.com/ansible/vscode-ansible)
+
+## Scaffolding collection using ansible-creator
+
+- Open VS Code and click the Ansible icon in the activity bar to access Ansible Creator section. Click on "Get Started" under that section to open the menu page of Ansible Creator in VS Code.
+
+- Check system requirements and install ansible-creator if needed. Ensure all requirements in the `system requirements` box have green ticks.
```

Comment:
Perhaps link to the installation page? https://ansible.readthedocs.io/projects/dev-tools/installation/